ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py

```diff
@@ -1294,7 +1294,7 @@ def test_numpy_ndarray_fill(
         available_dtypes=helpers.get_dtypes("numeric"), num_arrays=2
     ),
 )
-def test_numpy_instance_add__(
+def test_numpy_add__(
```

Comment:
Same thing in this file. The easiest way to not get confused is to copy the name from `method_name` above.

Reply:
Done